Tests/test_file_xvthumb.py

```diff
@@ -1,13 +1,35 @@
-from helper import unittest, PillowTestCase
+from helper import hopper, unittest, PillowTestCase
 
-from PIL import XVThumbImagePlugin
+from PIL import Image, XVThumbImagePlugin
+
+TEST_FILE = "Tests/images/hopper.p7"
 
 
 class TestFileXVThumb(PillowTestCase):
 
+    def test_open(self):
+        # Act
+        im = Image.open(TEST_FILE)
+
+        # Assert
+        self.assertEqual(im.format, "XVThumb")
+        self.assert_image_similar(im, hopper("P"), 49)
```

Comment:
`im` looks like this:
![image](https://cloud.githubusercontent.com/assets/1324225/23608980/8d259d6e-0274-11e7-9cd5-a0e01a1485cf.png)